yarn.lock

```diff
@@ -5406,7 +5406,7 @@ rc@^1.1.7:
     minimist "^1.2.0"
     strip-json-comments "~2.0.1"
 
-react-dom@^16.3.0:
+react-dom@^16.8.0:
```

Comment:
lol whoops

Reply:
should I remove these changes?

Reply:
nah these are good, i just didn't update it when i updated package requirements